sql/dd/impl/upgrade/server.cc

```diff
@@ -560,6 +560,15 @@ static bool check_tables(THD *thd, std::unique_ptr<Schema> &schema,
   auto process_table = [&](std::unique_ptr<dd::Table> &table) {
     invalid_triggers(thd, schema->name().c_str(), *table);
 
+    // The TokuDB engine was removed in 8.0.28 Don't upgrade if it is used.
+    if (my_strcasecmp(system_charset_info, table->engine().c_str(), "TokuDB") ==
```

Comment:
can we use db_type check with DB_TYPE_TOKUDB?

Reply:
@satya-bodapati , I'd prefer to be consistent which the way the engine is check in the rest of the file. Please, mind how some lines below, how the check for the engine "InnoDB" is made

Reply:
Fine with me.